src/pages/docs/support/debug-problems-with-octopus-variables.md

```diff
@@ -9,7 +9,7 @@ navOrder: 1
 
 Sometimes a variable used during deployment may have a different value from the one you expect. Here are the first steps to debugging these issues.
 
-## Check the variable snapshot for the release {#DebugproblemswithOctopusvariables-ChecktheVariableSnapshotfortheRelease}
+## Check the variable snapshot for the release {#check-variable-snapshot-for-release}
```

Comment:
I followed the same approach Steve Fenton used in this previous PR where we encountered this: https://github.com/OctopusDeploy/docs/pull/2360/commits/2bd1155bb6a346f4ff1819565f39c3bc515f9a70

And confirmed there's no anchor link references to these that needed updating.